core/src/cluster_info.rs

```diff
@@ -1805,10 +1805,11 @@ mod tests {
 
     #[test]
     fn new_replicator_external_ip_test() {
-        let ip = IpAddr::V4(Ipv4Addr::from(0));
+        let ip = Ipv4Addr::from(0);
         let node =
-            Node::new_replicator_with_external_ip(&Keypair::new().pubkey(), &socketaddr!(0, 8050));
+            Node::new_replicator_with_external_ip(&Keypair::new().pubkey(), &socketaddr!(ip, 0));
```

Comment:
the other test that uses 8050 should probably be changed, too...

Reply:
oh, to what? something outside fullnode_port_range?

Reply:
my thinking is to go bind a random port, then pass that number in.  tests that stake out global ports interfere with each other

Reply:
Ok; take a look.